crud.py

```diff
@@ -0,0 +1,67 @@
+from sqlalchemy.orm import Session
+
+import schemas
+from db import models
+from db.models import Author, Book
+
+
+def get_all_authors(
+        db: Session,
+        skip: int = 0,
+        limit: int = 10
+) -> list[Author] | None:
```

Comment:
```suggestion
) -> list[Author]:
```
It is impossible, it always return list